README.rst

```diff
@@ -40,14 +40,19 @@ Or with `pre-commit <https://pre-commit.com/>`__ in the ``repos`` section of you
         hooks:
         -   id: django-upgrade
             args: [--target-version, "4.1"]   # Replace with Django version
+            types: [file, python]
```

Comment:
this is already in the hook spec: https://github.com/adamchainz/django-upgrade/blob/main/.pre-commit-hooks.yaml#L6

And I want to keep it there - since we'll add support for template files at some point..

Also `file` is also not needed - it's not possible for a 'python directory' in identify - `python` is based on extension, which is only detected for files:

https://github.com/pre-commit/identify/blob/8a6ab8dd14c708770a999b76d2b929af06266c1b/identify/identify.py#L40-L64

Reply:
Great, will remove! I somehow missed the line :facepalm: